### PR TITLE
Bug hunting 2

### PR DESCRIPTION
**Pull request by Stearnzy**
Rubocop sweep; many files altered!!

Refactored some long methods/actions into helper methods
Added dependent: :destroy relations to Movie and User Models
  * I think these are correct, but I'm not 100% sure.
  * No tests written for these yet since there is no functionality for deleting a user, and I'm not sure how to test for an API resource being deleted.

4 Rubocop violations left, will ask in check-in how to approach.
Test coverage still at 100% withholding the 4 app files at 0%

### CODE REVIEW

File: app/controllers/friendships_controller.rb [hunk 1]
```diff
@@ -1,16 +1,19 @@
 class FriendshipsController < ApplicationController
   def create
     friend = User.find_by(email: params[:email])
-    unless friend.nil?
-      friendship = Friendship.find_by(user_id: current_user.id, friend_id: friend.id)
+    if friend.nil? || current_user.friends.include?(friend)
```
Comment (stillsheryl) [inline app/controllers/friendships_controller.rb:4]: Great refactoring!

File: app/poros/movie_object.rb [hunk 2]
```diff
@@ -28,7 +27,7 @@ def runtime_conversion
     hours = @runtime / 60
     min = @runtime % 60
 
-    if hours == 0
+    if hours.zero?
```
Comment (stillsheryl) [inline app/poros/movie_object.rb:30]: Cool, I didn't know this was a method.

**Review by stillsheryl — APPROVED** [review]
Thanks, Zach and Rubocop! :)

File: app/poros/movie_object.rb (hunk 2 above)
Comment (Eternal-Flame085) [reply app/poros/movie_object.rb:30]: nice thing to know

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -24,11 +21,11 @@ def destroy
   private
 
   def error_message
-    if params[:password].blank? && params[:email].blank?
-      flash[:error] = 'Fields cannot be blank.'
-    else
-      flash[:error] = 'Invalid email or password.'
-    end
+    flash[:error] = if params[:password].blank? && params[:email].blank?
```
Comment (Eternal-Flame085) [inline app/controllers/sessions_controller.rb:24]: did not know it could be done like this

File: app/views/dashboard/index.html.erb [hunk 4]
```diff
@@ -20,7 +20,7 @@
 </section>
 
 <section id='Viewing_Parties'>
-  <h2>Viewing_Parties</h2>
+  <h2>Viewing Parties</h2>
```
Comment (Eternal-Flame085) [inline app/views/dashboard/index.html.erb:23]: Good catch

**Review by Eternal-Flame085 — APPROVED** [review]

**Pull request merged**